src/sync/persist-fixup-transformer.test.ts

```diff
@@ -0,0 +1,360 @@
+import {assert} from '@esm-bundle/chai';
```

Comment:
wondering why `assert` rather than `expect` as we use in most tests.

Reply:
Someone was complaining about expect and I wanted to get  a feeling for assert

Reply:
my only gripe with using assert atm is that `assert` does not do TS assertions which I've used in a few test places.